other/bootstrap_daemon/tox_bootstrap.service

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=Tox DHT Bootstrap Daemon
+After=network.target
+
+[Service]
+Type=forking
+PermissionsStartOnly=true
+ExecStartPre=/usr/bin/touch /var/run/tox-bootstrapd.pid
+ExecStartPre=/usr/bin/chown tox-bootstrapd:tox-bootstrapd -R /var/run/tox-bootstrapd.pid
```

Comment:
Fixed the commands a little bit. If you are reading from email, then you won't see them fixed.